plugins/README.md

```diff
@@ -0,0 +1,37 @@
+# Git Theta Plug-ins
+
+Git theta support plugins for custom model checkpoint formats.
+
+## Writing a Checkpoint Plug-in.
+
+A checkpoint plug-in should subclass `git_theta.checkpoints.Checkpoint`.  The plugin
+should implement the `_load` method which reads the checkpoint format into a dict
```

Comment:
You changed the name to `load` right? Or am I looking in the wrong place?

Reply:
Updated.